ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java

```diff
@@ -37,11 +36,11 @@
 @Singleton
 public class NdbenchAWSCredentialProvider implements AWSCredentialsProvider {
 
-    private DynamoDBConfigs config;
+    private IDynamoDBConfiguration config;
```

Comment:
I believe that  Hungarian style notation has been deprecated.

Reply:
I saw an example of it in other places in ndbench. if you want me to revert, I will.

Reply:
for example https://github.com/Netflix/ndbench/blob/master/ndbench-janusgraph-plugins/src/main/java/com/netflix/ndbench/plugin/janusgraph/configs/IJanusGraphConfig.java

Reply:
That is a good point. I suggest we move towards the correct way. If you want to change that configuration to the proper notation in this PR, I am fine merging it. All the rest look good.

Reply:
I think we can change the notation in a different PR. Ill revert on this one

Reply:
fixed